docs/tech-reference/iso-rsync-distributor.rst

```diff
@@ -57,8 +57,8 @@ The ``distributor_config`` contains a ``remote`` section with the following sett
 
 ``root``
   The absolute path to the remote root directory where all the data (content and published content)
-  lives. This is the remote equivalent to ``/var/lib/pulp``. The repo id is appended to the
-  ``root`` path to determine the location of published repository.
+  lives. This is the remote equivalent to ``/var/lib/pulp``. The repository's relative url is
```

Comment:
The ISO Rsync distributor requires a predistributor. The only predistributor supported is an ISO distributor. The ISO distributor's relative_url is useb by the ISO Rsync distributor. I'll update my PR to make this more clear.